Packages/Tracking/Core/Editor/Scripts/HandPoseEditorScripts.cs

```diff
@@ -0,0 +1,156 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEditor.PackageManager.UI;
+using UnityEngine;
+
+
+namespace Leap.Unity.HandsModule
+{
+    [CustomEditor(typeof(HandPoseRecorder))]
+    public class HandPoseRecoderEditor : Editor
+    {
+        public override void OnInspectorGUI()
+        {
+            DrawDefaultInspector();
+
+            HandPoseRecorder poseRecorderScript = (HandPoseRecorder)target;
+            if (GUILayout.Button("Save Current Hand Pose"))
+            {
+                poseRecorderScript.SaveCurrentHandPose();
+            }
+        }
+    }
+
+    [CustomEditor(typeof(HandPoseScriptableObject))]
+    public class HandPoseSerializableObjectEditor : Editor
+    {
+        float _boneThresholdSlider = 15f;
+        bool _showFineTuningOptions = false;
+
+        public override void OnInspectorGUI()
+        {
+            EditorGUILayout.LabelField("Angle of tolerance for pose");
+
+            _boneThresholdSlider =  EditorGUILayout.Slider(_boneThresholdSlider, 0f, 90f);
+            HandPoseScriptableObject serializedObjectScript = (HandPoseScriptableObject)target;
+            serializedObjectScript.SetAllBoneThresholds(_boneThresholdSlider);
+
+            GUILayout.Space(15);
+
+            if (GUILayout.Button("Show Fine Tuning Options"))
+            {
+                _showFineTuningOptions = !_showFineTuningOptions;
+            }
+            
+            if (_showFineTuningOptions)
+            {
+                DrawDefaultInspector();
+            }
+            
+        }
+    }
+
+    [CustomEditor(typeof(HandPoseDetector))]
+    public class PoseDetectionEditor : Editor
+    {
+        Vector2 scrollPosition;
+
+        bool _showFineTuningOptions = false;
+        public override void OnInspectorGUI()
+        {
+            HandPoseDetector poseDetectionScript = (HandPoseDetector)target;
```

Comment:
To also show the 'script' section at the top when using this custom editor, you can use:

> EditorGUI.BeginDisabledGroup(true); 
> EditorGUILayout.ObjectField("Script", MonoScript.FromMonoBehaviour((MonoBehaviour)target), GetType(), false);
> EditorGUI.EndDisabledGroup();